src/containers/GameContainer.ts

```diff
@@ -1,14 +1,22 @@
+import _ from 'lodash';
 import { connect } from 'react-redux';
 
 import { GameAction } from '../redux/actions/game';
 import { OpponentAction } from '../redux/actions/opponents';
 import GameController from '../components/GameController';
 import { ApplicationState } from '../redux/reducers';
 
-const mapStateToProps = (state: ApplicationState) => ({
-  applicationState: state.game,
-  opponents: state.opponents,
-});
+const mapStateToProps = (state: ApplicationState) => {
+	return {
+	  applicationState: state.game,
+	  opponents: getOpponents(),
+	}
+
+	function getOpponents() {
+		const currentUserAddress = _.get(state, 'login.wallet.account.address');
```

Comment:
Once we have things typed properly, we could change this to directly access state.login.wallet.account.address.  Too bad `wallet` is `any` :(